scripts/terraform-fmt.ts

```diff
@@ -0,0 +1,66 @@
+import { exec, ExecException } from 'child_process';
```

Comment:
WDYT about rewriting this using https://github.com/google/zx ?

Reply:
I mean, I can but I don't really see any additional value, especially for such a small linting script :man_shrugging:

Reply:
I really like that package and how concise it is. I think it would be a good idea to start using it even for such a small script - but up to you. I do not feel strongly about this.

Reply:
I would lean towards keeping this as a regular TS script

Reply:
> WDYT about rewriting this using https://github.com/google/zx ?

This is very cool, I might use this for personal stuff.